xbmc/storage/MediaManager.cpp

```diff
@@ -59,6 +61,8 @@ using namespace XFILE;
 using namespace MEDIA_DETECT;
 #endif
 
+using namespace KODI::MESSAGING;
```

Comment:
Im not sure its worth adding a using namespace for a single line.

Just my opinion though, if anyone else has a preference, im easy

Reply:
i'm fine changing this